installation/resources/installer-config-cluster.yaml.tpl

```diff
@@ -67,6 +67,22 @@ data:
   gateways.istio-ingressgateway.loadBalancerIP: "__EXTERNAL_PUBLIC_IP__"
   global.proxy.excludeIPRanges: "__PROXY_EXCLUDE_IP_RANGES__"
   global.tracer.zipkin.address: zipkin.kyma-system:9411
+
```

Comment:
>  A fair amount of those resources was used by istio.

What exactly do you mean? which components and how much do they use? 
Pilot by default has  memory request set to 2gig, have you tested it on 1 gig limit? 
The same with policy, generally I'm against setting the same limit as for minikube

Reply:
Second thing, cluster values should be in the charts. We aim to get rid of this cluster config file.